hs_core/tests/api/native/test_resource_file.py

```diff
@@ -0,0 +1,330 @@
+import os
+
+from django.test import TransactionTestCase
+from django.contrib.auth.models import Group
+from django.core.exceptions import ValidationError
+
+from hs_core import hydroshare
+from hs_core.testing import MockIRODSTestCaseMixin, TestCaseCommonUtilities
+
+from hs_core.models import ResourceFile, get_path
+
+
+class TestResourceFileAPI(MockIRODSTestCaseMixin,
+                          TestCaseCommonUtilities, TransactionTestCase):
+    def setUp(self):
+        super(TestResourceFileAPI, self).setUp()
+        self.hydroshare_author_group, _ = Group.objects.get_or_create(name='Hydroshare Author')
+        # create a user to be used for creating the resource
+        self.user = hydroshare.create_account(
+            'creator@usu.edu',
+            username='creator',
+            first_name='Creator_FirstName',
+            last_name='Creator_LastName',
+            superuser=False,
+            groups=[]
+        )
+
+        self.res = hydroshare.create_resource(
+            'GenericResource',
+            self.user,
+            'My Test Resource'
+        )
+
+        # create a file
+        self.test_file_name1 = 'file1.txt'
+        # self.test_file_name2 = 'file2.txt'
```

Comment:
I assume these commented code statements here in other places in this file can be removed.